app/dependencies.py

```diff
@@ -15,15 +15,27 @@
 TEMPLATES_PATH = os.path.join(APP_PATH, "templates")
 SOUNDS_PATH = os.path.join(STATIC_PATH, "tracks")
 templates = Jinja2Templates(directory=TEMPLATES_PATH)
-templates.env.add_extension('jinja2.ext.i18n')
+templates.env.add_extension("jinja2.ext.i18n")
 
 # Configure logger
-logger = LoggerCustomizer.make_logger(config.LOG_PATH,
-                                      config.LOG_FILENAME,
-                                      config.LOG_LEVEL,
-                                      config.LOG_ROTATION_INTERVAL,
-                                      config.LOG_RETENTION_INTERVAL,
-                                      config.LOG_FORMAT)
+logger = LoggerCustomizer.make_logger(
+    config.LOG_PATH,
+    config.LOG_FILENAME,
+    config.LOG_LEVEL,
+    config.LOG_ROTATION_INTERVAL,
+    config.LOG_RETENTION_INTERVAL,
+    config.LOG_FORMAT,
+)
+
+if os.path.isdir(config.UPLOAD_DIRECTORY):
+    UPLOAD_PATH = config.UPLOAD_DIRECTORY
+else:
+    try:
+        UPLOAD_PATH = os.path.join(os.getcwd(), config.UPLOAD_DIRECTORY)
+        os.mkdir(UPLOAD_PATH)
+    except OSError as e:
+        logger.critical(e)
+        raise OSError(e)
```

Comment:
@yammesicka this is the check I added in case the user prefers for testing to create the folder under app.